src/plots/cartesian/graph_interact.js

```diff
@@ -309,6 +309,13 @@ fx.unhover = function (gd, evt, subplot) {
 // The actual implementation is here:
 
 function hover(gd, evt, subplot){
+    if(gd._fullLayout._hasPie){
```

Comment:
@mdtusz On second thought, a check on `_hasPie` is too coerced.

Say we have a pie and bar trace on the same graph:

```js
var gd = Tabs.fresh();

Plotly.plot(gd, 
    [
        {
            type: 'pie',
            labels: ['a', 'b', 'c'],
            values: [1, 2, 3],
            domain: {
                x: [0, 0.45]
            }
        },
        {
            type: 'bar',
            x: ['A', 'B', 'C'],
            y: [1, 2, 3],
        }
    
    ],
    {
        title: 'Pie vs bar',
        xaxis: { domain: [0.55, 1] },
        showlegend: false
    }
);

gd.on('plotly_hover', function(data) {
    var pt = data.points[0]
    console.log(pt);
});
```

then hovering over the bars will result in an un-formatted hover data.

So, we should use the `subplot` argument and make the if-clause: `if(subplot === 'pie')`.